packages/server/src/auth/CrossmintAuth.ts

```diff
@@ -20,4 +30,62 @@ export class CrossmintAuth {
     public static from(crossmint: Crossmint): CrossmintAuth {
         return new CrossmintAuth(crossmint);
     }
+
+    public getJWKSUri() {
+        return `${this.apiClient.baseUrl}/.well-known/jwks.json`;
+    }
```

Comment:
is this something that needs to be public/expected to be used from a client? Or more meant as the helper for `verifyCrossmintJwtToken`?

Reply:
It's different levels of utilities to access it:

1. The actual endpoint
2. This function to easily get the endpoint
3. We then use it internally if they trust our implementation of the verification, but they can also verify themselves